esi_leap/db/sqlalchemy/api.py

```diff
@@ -253,7 +253,8 @@ def contract_get_all_by_status(context, status):
 def contract_create(context, values):
     contract_ref = models.Contract()
     values['uuid'] = uuidutils.generate_uuid()
-    values['project_id'] = context.project_id
+    # modified: project_id for a contract should be the offer project_id??
+    # values['project_id'] = context.project_id
```

Comment:
I changed the code here, because I think the contract project_id should be the offer project_id. Correct me if I'm wrong.

Reply:
Not as currently written; the contract project_id should be set to the marketplace bid's project_id, as that is the value that is needed to give access to a consumer. It's possible that the column should be renamed long-term, but for now I think you're right to remove this default.

Reply:
We should probably just remove these commented out lines.